lib/lens/core.rb

```diff
@@ -4,9 +4,4 @@
 require "lens/event_formatter"
 require "lens/gc"
 require "lens/trace"
-
-module Lens
-  if defined?(Rails)
-    require "lens/railtie"
-  end
-end
+require "lens/worker"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.